src/lib/draw/point_tool.ts

```diff
@@ -1,21 +1,29 @@
-import { emptyGeojson } from "../../stores.js";
+import type { Feature, Point } from "geojson";
+import type { Map, GeoJSONSource } from "maplibre-gl";
 import {
+  emptyGeojson,
   overwriteSource,
   overwriteLayer,
   drawCircle,
 } from "../../maplibre_helpers";
-import { colors, circleRadius } from "../../colors.js";
+import { colors, circleRadius } from "../../colors";
 
+const source = "edit-point-mode";
+
+// Note this uses the geojson Feature, not our specialization in types.ts
 export class PointTool {
+  map: Map;
```

Comment:
This kind of thing brings me great comfort, just seeing all the state and types upfront